tests/test_run_service.py

```
@@ -535,6 +535,7 @@ def test_list_runs_paginated_with_flow_key_filter(self, tmp_path, monkeypatch):
         monkeypatch.setattr(storage, "list_runs", lambda runs_dir=None: run_ids)
         monkeypatch.setattr(storage, "discover_example_runs", lambda: [])
         monkeypatch.setattr(storage, "discover_legacy_runs", lambda runs_dir=None: [])
+        monkeypatch.setattr(storage, "scan_runs", lambda runs_dir=None: (run_ids, []))
```

Ruff (0.15.12)

[warning] Unused lambda argument: `runs_dir`

(ARG005)